### PR TITLE
Bug 1250801 - [GijTV] Add permission_manager_test.js in smart-system

### PR DESCRIPTION
**Pull request by Fischer-L**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=701a4d0ee4c11728ae56290790ee0d2578f3e361

File: tv_apps/smart-system/test/marionette/permission_manager_test.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  setup(function() {
+    system = client.loader.getAppClass('system');
+    system.waitForStartup();
+    system.waitForFullyLoaded();
+    system.waitForLaunch(PermissionManager.APP_URL);
+    permissionManager = new PermissionManager(client);
+    client.switchToFrame();
+  });
+
+  test('Should open', testOptions, function () {
```
Comment (luke-chang) [inline tv_apps/smart-system/test/marionette/permission_manager_test.js]: nit: `'Should open and close'`

File: tv_apps/smart-system/test/marionette/permission_manager_test.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    permissionManager.yesBtn.scriptWith(function(yesBtn) {
+      return document.activeElement === yesBtn;
+    }, function (err, focused) {
+      assert.ok(focused, 'Not focus on yes button');
+    });
+    permissionManager.yesBtn.sendKeys(Keys.enter);
+    permissionManager.waitForPermissionDialogClosed();
+  });
+
+  test('Should display more info', testOptions, function () {
```
Comment (luke-chang) [inline tv_apps/smart-system/test/marionette/permission_manager_test.js]: nit: `'Should toggle more info'`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5e8aa23d807cf55df816231282ef74c918bb4386

**Pull request merged**